test/common/proto_utils_test.cpp

```diff
@@ -14,16 +14,16 @@ TEST(ProtoUtilsTest, MachineIdStringToMachineId) {
   internal::MachineId mid;
 
   mid = MakeMachineId("0:0");
-  ASSERT_EQ(mid.replica(), 0);
-  ASSERT_EQ(mid.partition(), 0);
+  ASSERT_EQ(mid.replica(), 0U);
```

Comment:
What is this change?

Reply:
It is to get rid of warnings when we compare unsigned integers with signed integers (literals are signed by default)